README.md

```diff
@@ -25,14 +25,19 @@ Note that there is currently no separate development environment for mixpanel, s
     ```sh
     yarn install
     ```
-
-3. Start a dev server on port 8080 with auto-reload
+3. To run bard locally you need to create a config.json in the root directory as there is no specified default config.
+   From within your local bard repo run the following command in the terminal:
+   
+   ```sh
+   cp config/dev.json config.json
+   
```

Comment:
does this need a ``` below it just to be syntactically like the other lines?

Reply:
oof good call out, that's what I get for going too-fast-too-furious on this PR haha